src/v2.rs

```diff
@@ -223,6 +224,17 @@ impl Options {
   }
 }
 
+pub struct FromGraphOptions<'a> {
+  pub graph: ModuleGraph,
+  pub parser: CapturingModuleParser<'a>,
+  pub transpile_options: TranspileOptions,
+  pub emit_options: EmitOptions,
+  /// Base to make all descendant file:/// modules relative to.
+  ///
+  /// Note: When a path is above the base it will be left absolute.
```

Comment:
What are the implications of this scenario?

Reply:
See the test: from_graph_relative_base_windows_different_drives